src/control/server/mgmt_svc.go

```diff
@@ -41,25 +40,6 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
-// CheckReplica verifies if this server is supposed to host an MS replica,
-// only performing the check and printing the result for now.
-func CheckReplica(lis net.Listener, accessPoints []string, srv *exec.Cmd) (msReplicaCheck string, err error) {
```

Comment:
No longer used?

Reply:
yes dead code, mentioned in the commit message